datafusion/core/src/physical_planner.rs

```diff
@@ -2540,6 +2540,27 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn aggregate_with_alias() -> Result<()> {
```

Comment:
I found a better place for this test (with the other tests that are checking things about the physical plans)

Reply:
Looks good. 👍